### ISSUE TITLE
[Adaptive Cards - ExpenseReport.Json]: Screen Reader does not announce the name and state of the collapse down arrow icon buttons present under 'Amount' under 'Expense approval card'.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6979]
https://microsoft.visualstudio.com/OS/_workitems/edit/35357132
